airbyte-integrations/connectors/source-amplitude/source_amplitude/api.py

```diff
@@ -231,7 +231,13 @@ def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapp
         if response_data:
             series = list(map(list, zip(*response_data["series"])))
             for i, date in enumerate(response_data["xValues"]):
-                yield {"date": date, "statistics": dict(zip(response_data["seriesLabels"], series[i]))}
+                try:    
+                    yield {"date": date, "statistics": dict(zip(response_data["seriesLabels"], series[i]))}
+                except IndexError as e:
+                    # To handle index out of range exception being thrown
+                    self.logger.exception(e)
+                except KeyError as e:
+                    self.logger.exception(e)
```

Comment:
```suggestion
                except (IndexError, KeyError) as e:
                    self.logger.exception(e)
```
https://docs.python.org/3/tutorial/errors.html#handling-exceptions

Reply:
Done